pom.xml

```diff
@@ -102,6 +103,22 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <requireUpperBoundDeps>
+              <excludes combine.children="append">
+                <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
+                <exclude>org.apache.ant:ant</exclude>
+                <!-- structs requests 1.9, core requests 1.7 -->
+                <exclude>org.jenkins-ci:annotation-indexer</exclude>
+                <!-- mockito requests 2.1, powermock requests 2.4 -->
+                <exclude>org.objenesis:objenesis</exclude>
+                <!-- workflow-cps requests 1.1.1, core 2.60.1 requests 1.2.1 -->
+                <exclude>org.jenkins-ci.ui:jquery-detached</exclude>
```

Comment:
> But you cannot use that while keeping your ancient core baseline.

I don't currently plan to update the baseline until the switch to 2.60.1 as the next baseline.  I assume that will be later this year, and will be the git plugin 4 release, with ongoing lightweight maintenance of the git plugin 3.x releases.  I think it should follow the same "old versions die slowly" model that was used with git plugin 2.x in the transition to git plugin 3.x.

Once the baseline is 2.60.1 or newer, the git client plugin can switch from JGit 4.5 to a more recent JGit release (4.8+).  I haven't yet found other compelling gains from requiring JDK 8 runtime, but I assume there are other benefits as well.